contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1215,6 +1236,189 @@ pre_transform_target_entry(ResTarget *res, ParseState *pstate,
 			res->name = alias;
 		}
 	}
+	/* Update table set qualified column name, resolve qualifiers here */
```

Comment:
This is column name handling in APG, controlled by `ansi_qualified_update_set_target`.